.github/workflows/publish_docker_images.yml

```diff
@@ -31,6 +32,7 @@ jobs:
             type=edge,branch=main,suffix=-bullseye
             type=semver,pattern={{version}}
             type=semver,pattern={{version}},suffix=-bullseye
+            type=ref,event=tag
```

Comment:
Just be aware that if you add a `happy-plazza` git tag that is not at the tip of `main`, we're (temporarily) moving the `edge` Docker tag back.

Reply:
Mmm, yes I'm discovering the metadata action. We can avoid that with

```
enable=${{ !startsWith(github.ref, 'refs/tags') }}
```

`edge` tag should be triggered only on a commit push on main.

Reply:
I'm quite new on this, what do you think?

Reply:
I'm rereading the doc of the metadata action, and it might be smart enough to push an edge tag only for the last commit. Let's try this as is.